models/Asset.php

```diff
@@ -727,7 +728,11 @@ protected function update($params = [])
 
                 $this->closeStream(); // set stream to null, so that the source stream isn't used anymore after saving
 
-                $mimeType = $storage->mimeType($path);
+                try {
+                    $mimeType = $storage->mimeType($path);
+                } catch(UnableToRetrieveMetadata $e) {
+                    $mimeType = 'unknown';
```

Comment:
```suggestion
                    $mimeType = 'application/octet-stream';
```
We should use `application/octet-stream` here instead, because `unknown` is not a valid mime-type, right?